src/components/singleton-editor/useSingletonEditor.ts

```diff
@@ -24,6 +26,9 @@ export const useSingletonEditor = ({
 }) => {
   const { unlayerEditorObject, setContent } = useSingletonDesignContext();
 
+  // TODO: enable it when CustomMediaLibrary component be ready
+  // useCustomMediaLibrarySetup({ unlayerEditorObject });
+
```

Comment:
This is the code that we should to uncomment in order to enable the new behavior.